aten/src/ATen/core/ivalue_inl.h

```diff
@@ -748,6 +772,10 @@ inline IValue::IValue(c10::intrusive_ptr<ivalue::Object> v)
 : tag(Tag::Object), is_intrusive_ptr(true) {
   payload.as_intrusive_ptr = v.release();
 }
+inline IValue::IValue(c10::intrusive_ptr<ivalue::PythonObject> v)
```

Comment:
We should have a version of the constructor that takes a PyObject or py::object directly and is defined somewhere in libtorch_python.so